src/gen_dbus.erl

```diff
@@ -31,11 +28,8 @@
 	 terminate/2
 	]).
 
-behaviour_info(callbacks) ->
-    [
-     {init, 1},
-     {handle_info, 2}
-    ].
+-callback init(tuple()) -> atom().
+-callback handle_info(tuple(), term()) -> atom().
```

Comment:
Made this more correct. the officail type spec for handle_Info is: 

[Module:handle_info(Info, State) -> Result]()

Types:

Info = timeout | term()
State = term()
Result = {noreply,NewState} | {noreply,NewState,Timeout}
  | {noreply,NewState,hibernate}
  | {stop,Reason,NewState}
 NewState = term()
 Timeout = int()>=0 | infinity
 Reason = normal | term()

Reply:
cool :+1: